ddtrace/ext/test.py

```diff
@@ -38,6 +38,9 @@
 # Test Type
 TYPE = TEST_TYPE = "test.type"
 
+# Test File
```

Comment:
will this be the location of the `.feature` file?

Reply:
Yes

Reply:
No. The `.feature` file will be in `test.suite`. This is a new tag for step location. Maybe I should move it to `ddtrace.contrib.pytest_bdd.constants or ext`.